src/compiler/types.ts

```diff
@@ -2870,6 +2870,7 @@ namespace ts {
     /* @internal */
     export interface TransientSymbol extends Symbol, SymbolLinks {
         checkFlags: CheckFlags;
+        isRestParameter?: boolean; 

```

Comment:
This seems kind of invasive for a small change, and the property is only checked one place. Is it possible to use some other predicate in buildParameterDisplay? Would `isTransientSymbol` be enough?